backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala

```diff
@@ -715,6 +716,21 @@ trait StandardAsyncExecutionActor
     case _ => 0
   }
 
+  /**
+   * Returns the memory multiplier for previous attempt if available
+   */
+  lazy val previousMemoryMultiplier: Option[MemoryRetryMultiplierRefined] = jobDescriptor.prefetchedKvStoreEntries.get(BackendLifecycleActorFactory.MemoryMultiplierKey) match {
+    case Some(KvPair(_,v)) => refineV[MemoryRetryMultiplier](v.toDouble) match {
+      case Left(e) =>
+        // should not happen as Cromwell itself had written the value as a Double
+        log.error(e, s"Programmer error: unexpected failure attempting to read value for MemoryMultiplierKey from Key Value table. " +
+          "Expected value should be in range 1.0 ≤ n ≤ 99.0")
+        None
+      case Right(refined) => Option(refined)
+    }
+    case _ => None
```

Comment:
Is it possible that this would be `Some(somethingElse)`, or is this `case _` equivalent to `case None => None`?

And typing out `case None => None` has just made me wonder... Is there a reason for using `case`/`match` here rather than expressing this as a `flatMap`?

Reply:
Yes. `case _` is intended to cover the other cases mentioned below. Converting it into a `flatMap` gives the below error:
```
It would fail on the following inputs: KvFailure(_, _), KvKeyLookupFailed(_), KvPutSuccess(_) 
```

Reply:
like this?
```
lazy val previousMemoryMultiplier: Option[Double] = jobDescriptor.prefetchedKvStoreEntries.get(BackendLifecycleActorFactory.MemoryMultiplierKey) flatMap {
    case KvPair(_,v) => Try(v.toDouble) match {
      case Success(m) => Option(m)
      case Failure(e) =>
        // should not happen as Cromwell itself had written the value as a Double
        log.error(e, s"Programmer error: unexpected failure attempting to convert value of MemoryMultiplierKey from JOB_KEY_VALUE_ENTRY table to Double.")
        None
    }
    case _ => None
```

Reply:
I guess that doesn't really cut down how much typing you have even if it _feels_ slightly nicer to me.

Happy for you to leave it as-is